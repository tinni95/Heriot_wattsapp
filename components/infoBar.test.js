import React from "react";
import renderer from "react-test-renderer";

import InfoBar from "./infoBar";

describe("infoBar", () => {
    const props = {
        style: {
            backgroundColor: "#E0A749",
            paddingBottom: 15,
        },
        number: "01314471135",
        openHour: 11,
        closingHour: 11,
    };
    let rendered;
    beforeEach(() => {
        rendered = renderer.create(<InfoBar {...props} />);
    });

    it("renders", () => {
        expect(rendered).toBeTruthy();
    });

    it("matches snapshot", () => {
        expect(rendered.toJSON()).toMatchSnapshot();
    });
});
