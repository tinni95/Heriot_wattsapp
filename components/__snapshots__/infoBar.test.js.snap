// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`infoBar matches snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0A749",
      "paddingBottom": 15,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingLeft": 8,
        "paddingTop": 40,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "marginRight": 20,
        }
      }
    >
      <View
        style={
          Object {
            "marginRight": 10,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
          underlayColor="white"
        >
          <Text />
        </View>
      </View>
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 14,
          }
        }
      >
        01314471135
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "marginRight": 20,
        }
      }
    >
      <View
        style={
          Object {
            "marginRight": 10,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
          underlayColor="white"
        >
          <Text />
        </View>
      </View>
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 14,
          }
        }
      >
        open: 11 am-11 pm
      </Text>
    </View>
  </View>
</View>
`;
